create default admin cause cannot add indexers to running index

cannot add indexers to running index of the default admin
```
time="2020-11-02T02:52:31Z" level=info msg="Listening on :8443"
time="2020-11-02T02:52:32Z" level=info msg="Starting harvester.cattle.io/v1alpha1, Kind=VirtualMachineImage controller"
time="2020-11-02T02:52:32Z" level=info msg="Starting /v1, Kind=Secret controller"
time="2020-11-02T02:52:32Z" level=info msg="Active TLS secret serving-cert (ver=1209280) (count 6): map[listener.cattle.io/cn-10.42.0.155:10.42.0.155 listener.cattle.io/cn-10.42.0.95:10.42.0.95 listener.cattle.io/cn-10.42.3.13:10.42.3.13 listener.cattle.io/cn-10.42.3.9:10.42.3.9 listener.cattle.io/cn-10.42.4.11:10.42.4.11 listener.cattle.io/cn-172.16.0.63:172.16.0.63 listener.cattle.io/hash:b9226b4a0f99901e63c93d7eb6ea5b7067e5460e32552bdccd53945329c169fa]"
I1102 02:53:14.056000       7 leaderelection.go:252] successfully acquired lease kube-system/harvester-controllers
time="2020-11-02T02:53:14Z" level=error msg="error syncing 'user-gdj99': handler user-rbac-controller: Index with name auth.harvester.cattle.io/crb-by-role-and-subject does not exist, requeuing"
time="2020-11-02T02:53:14Z" level=error msg="error syncing 'user-gdj99': handler user-rbac-controller: Index with name auth.harvester.cattle.io/crb-by-role-and-subject does not exist, requeuing"
time="2020-11-02T02:53:14Z" level=info msg="Default admin already created, skip create admin step"
panic: cannot add indexers to running index

goroutine 2457 [running]:
github.com/rancher/harvester/vendor/k8s.io/apimachinery/pkg/util/runtime.Must(...)
	/go/src/github.com/rancher/harvester/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:171
github.com/rancher/harvester/vendor/github.com/rancher/wrangler-api/pkg/generated/controllers/rbac/v1.(*clusterRoleBindingCache).AddIndexer(0xc002bae1b0, 0x1af2d7a, 0x30, 0x1b7cc18)
	/go/src/github.com/rancher/harvester/vendor/github.com/rancher/wrangler-api/pkg/generated/controllers/rbac/v1/clusterrolebinding.go:239 +0x11d
github.com/rancher/harvester/pkg/indexeres.RegisterManagementIndexers(0xc000314380)
	/go/src/github.com/rancher/harvester/pkg/indexeres/indexer.go:22 +0xa4
github.com/rancher/harvester/pkg/controller/master.register(0x1d688a0, 0xc0010bca80, 0xc000134000, 0xc000314380, 0x0, 0x0)
	/go/src/github.com/rancher/harvester/pkg/controller/master/controller.go:39 +0xe6
github.com/rancher/harvester/pkg/controller/master.Setup.func1(0x1d688a0, 0xc0010bca80)
	/go/src/github.com/rancher/harvester/pkg/controller/master/setup.go:14 +0x54
created by github.com/rancher/harvester/vendor/github.com/rancher/wrangler/pkg/leader.run.func1
	/go/src/github.com/rancher/harvester/vendor/github.com/rancher/wrangler/pkg/leader/leader.go:58 +0x46
```

--- a/pkg/controller/master/controller.go
+++ b/pkg/controller/master/controller.go
@@ -26,6 +26,8 @@ var registerFuncs = []registerFunc{
 }
 
 func register(ctx context.Context, server *server.Server, management *config.Management) error {
+	indexeres.RegisterManagementIndexers(management)
+
 	for _, f := range registerFuncs {
 		if err := f(ctx, management); err != nil {
 			return err
@@ -36,6 +38,5 @@ func register(ctx context.Context, server *server.Server, management *config.Man
 		return err
 	}
 
-	indexeres.RegisterManagementIndexers(management)
 	return nil
 }

